docker-compose.yml

```diff
@@ -20,6 +20,7 @@ services:
 
   proxy:
     image: 'bahmni/proxy:${PROXY_IMAGE_TAG:?}'
+    profiles: ['snomed']
```

Comment:
Do we need a name like snomed? Also do we need a specific profile, as the default should do? we havent added any additional service.

Reply:
@angshu default profile bringing services like `odoo, crator, metabase etc`. so created this new profile `snomed` which brings only required services for this project.

Reply:
ah sorry .. I didnt realize this is against feature branch